changelog.d/12140.misc

```diff
@@ -0,0 +1 @@
+Move `synctl` into `synapse._scripts` and expose as an entry point.
```

Comment:
Might want to include a note in the upgrade notes, since server admins might be relying on synctl being where it is.

Reply:
That's done. @babolivier could you give it a quick sanity check please?